components/proxy/src/main/java/com/hotels/styx/admin/handlers/livenessreadiness/Liveness.java

```diff
@@ -0,0 +1,53 @@
+/*
+  Copyright (C) 2013-2019 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.admin.handlers.livenessreadiness;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import com.hotels.styx.api.HttpHandler;
+import com.hotels.styx.common.http.handler.HttpMethodHandler;
+
+import static com.google.common.net.MediaType.JSON_UTF_8;
+import static com.hotels.styx.api.HttpHeaderNames.CONTENT_TYPE;
+import static com.hotels.styx.api.HttpMethod.GET;
+import static com.hotels.styx.api.HttpResponse.response;
+import static com.hotels.styx.api.HttpResponseStatus.OK;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * Provides an endpoint for checking that the Styx JVM is running.
+ */
+public final class Liveness {
+    public static final HttpHandler LIVENESS_HANDLER = new HttpMethodHandler.Builder()
+            .add(GET, response(OK)
+                    .header(CONTENT_TYPE, JSON_UTF_8)
+                    .body(jsonBody(), UTF_8)
+                    .build())
+            .build();
+
+    private Liveness() {
+    }
+
+    private static String jsonBody() {
+        try {
+            return new ObjectMapper().writeValueAsString(
+                    ImmutableMap.of("live", "true"));
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+    }
```

Comment:
This always returns true? As such this liveness test is unnecessary because the requestor can probe any of the other URL endpoints regardless of the response.